multimodal/src/autogluon/multimodal/utils/object_detection_visualizer.py

```diff
@@ -0,0 +1,661 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# Disclaimer: Special thanks to the Detectron2 developers
+# https://github.com/facebookresearch/detectron2/blob/main/detectron2/utils/visualizer.py!
+# We use part of its provided, open-source functionalities.
+
+import colorsys
+import logging
+import math
+import numpy as np
+from enum import Enum, unique
+import cv2
```

Comment:
We need to put all `import cv2` inside functions. If users want to visualize, they will install opencv-python themselves. Otherwise cv2 will be a must-have dependency.

Reply:
that's right. Let me check on that. I believe this can be done in functions